### ISSUE TITLE
Path colors

### ISSUE THREAD

**Issue by thebuggy83** [Neop/mudmap2#39]
Hello!
I cannot change the default path colors, is that intended? 
I am using MacOSX Mojave and Java JDK 12.0.2 64 bit.
All other functions are working correctly. 
Is there a File where the colors are stored?
Thanks
BR Markus

**Comment by Neop** [comment]
Which version of mudmap are you using? I had been working on this issue a few days ago, it should be fixed in version 2.5.5.

The path colors are stored in each world file. The entries look like these (the last one is optional):
```
"pathCol":"#00ff97",
"pathColNonCardinal":"#ff9100",
"pathColDefs":[{"path":"sw","col":"#ff0000"},{"path":"ne","col":"#ff0000"},{"path":"nw","col":"#ff3800"}],
```

**Comment by thebuggy83** [comment]
color changing works in the new version thanks, but now place creation does not work properly, i always have to add a placeholder and change the place group to none.

**Comment by Neop** [comment]
Hmm, okay. I'll be able to look into it next week, I suggest to use the previous version till then.

**Comment by Neop** [comment]
I fixed the bug in version 2.5.6: https://github.com/Neop/mudmap2/releases/tag/v2.5.6
